src/psyclone/tests/gocean1p0_test.py

```diff
@@ -45,7 +45,8 @@
 from psyclone.parse.algorithm import parse
 from psyclone.psyGen import PSyFactory, InternalError
 from psyclone.generator import GenerationError, ParseError
-from psyclone.gocean1p0 import GOKern, GOLoop, GOInvokeSchedule
+from psyclone.gocean1p0 import GOKern, GOLoop, GOInvokeSchedule, \
+    GOKernelArgument, GOKernelArguments
```

Comment:
pylint complains that `GOKernelArgument` is subsequently re-imported further down.

Reply:
Deleted the duplicated import inside the test